glue-codes/fast-farm/LESinflow/NRELOffshrBsline5MW_Onshore_ElastoDyn_8mps.dat

```diff
@@ -73,6 +73,7 @@ False         PtfmYDOF    - Platform yaw rotation DOF (flag)
           0   TipMass(3)  - Tip-brake mass, blade 3 (kg) [unused for 2 blades]
       56780   HubMass     - Hub mass (kg)
      115926   HubIner     - Hub inertia about rotor axis [3 blades] or teeter axis [2 blades] (kg m^2)
+     115926   HubIner_Teeter     - Hub inertia about teeter axis [2 blades] (kg m^2)

```

Comment:
Does the description of line 75 need to change?  Maybe `Hub inertia about rotor axis [N blades]`?

Maybe also add on the `HubIner_Teeter`: `Hub inertia about teeter axis [only used for 2 blades with teeter] (kg m^2)`